Kudu.Services/Deployment/PushDeploymentController.cs

```diff
@@ -480,6 +481,13 @@ private async Task<HttpResponseMessage> PushDeployAsync(ArtifactDeploymentInfo d
 
             var response = Request.CreateResponse();
 
+            if (ArmUtils.IsArmRequest(Request))
+            {
+                DeployResult deployResult = _deploymentManager.GetDeployResult();
+                deployResult.Id = deploymentInfo.DeploymentTrackingId;
+                response = Request.CreateResponse(HttpStatusCode.Accepted, ArmUtils.AddEnvelopeOnArmRequest(deployResult, Request));
+            }
```

Comment:
This is a breaking change, correct?   Please put it under ScmHost... knob to allow safe rollout.